templates/eks/amazon-eks-nodepool-cf.yaml

```diff
@@ -11,8 +11,8 @@ Metadata:
         Parameters:
           - ClusterName
           - ClusterControlPlaneSecurityGroup
+          - CustomSecurityGroups
```

Comment:
Since we have parameters like `ClusterControlPlaneSecurityGroup`, I think keeping `Node` in `CustomNodeSecurityGroups` is a good idea.

Reply:
I agree, [renamed](https://github.com/banzaicloud/pipeline/compare/9752debc2564b091c0bb761a9d93b2f87a6c1a09..349b6830176c5819d348bd8035a7d53bf981a09e) all the corresponding references to contain `Node`.